sympy/geometry/point.py

```diff
@@ -233,21 +413,35 @@ def ambient_dimension(self):
 
     @property
     def orthogonal_direction(self):
-        """Returns a non-zero point in a direction
-        orthogonal to `self`."""
+        """Returns a non-zero point in the x-y plane that is
```

Comment:
You could have a line bewteen two 3D points. I'm not sure what a line between the origin and a 4D point looks like but consider:

```python
>>> p=Point(1,2,3,4)
>>> o=p.orthogonal_direction
>>> Line(Point(dim=4),p).angle_between(Line(Point(dim=4),o))
pi/2
```

(That's why I left it in the general Point class.)